src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/DefaultRazorSemanticTokenInfoServiceTest.cs

```diff
@@ -61,7 +61,25 @@ public void GetSemanticTokens_IgnoresNonTagHelperAttributes()
                 0, 39, 5, 0, 0
             };
 
-            AssertSemanticTokens(txt, expectedData);
+            AssertSemanticTokens(txt, expectedData, isRazor: false);
+        }
+
+        [Fact]
+        public void GetSemanticTokens_DoesNotFindCapitalizedTagHelpers()
```

Comment:
Wait is there a `TestElement` targeting TagHelper? If there is that should be highlighted.